conda_build_config.yaml

```diff
@@ -190,7 +190,7 @@ regex:
 rust_nightly:
   - 1.50.0
 scikit_image:
-  - 0.16
+  - 0.17
```

Comment:
I hope we have checked the main frameworks which use scikit_image with this version change. If yes, I'm okay with this.

Reply:
Yes only PyTorch uses it and I have checked the build on one flavour.